docs/Tutorial.md

```diff
@@ -198,4 +198,74 @@ Then you could use `Vec3` wherever you could use any other type:
 YAML::Node node = YAML::Load("start: [1, 3, 0]");
 Vec3 v = node["start"].as<Vec3>();
 node["end"] = Vec3(2, -1, 0);
+```
+
+## Partial specialization
+
+If you need to specialize the `convert` struct for a set of types instead of just one you can use partial specialization with the help of `std::enable_if` (SFINAE).
+
+Here is a small example showing how to partially specialize the `convert` struct for all types deriving from a base class:
+
+```cpp
+// Base class
+class A
```

Comment:
Inconsistent indentation. Match the format of new content with the  above.

Reply:
Fixed. Didn't realize it was tabs instead of spaces